docs/updating.rst

```diff
@@ -22,24 +22,28 @@ settings configured to your package and then pull the changes into your package.
    to make this process easier, and possibly even automated.
 
 
-1. **Render the template**: Putting in the same settings as the first time the
+#. **Render the template**: Putting in the same settings as the first time the
    template was rendered (or using ``--replay``) render the template to a
    temporary folder::
 
    $ cookiecutter [--replay] gh:astropy/package-template -o /tmp
 
-2. **Make a new branch**::
+#. **Make a new branch**::
 
    $ git checkout -b update_template
 
-3. **Copy the template over your package**: (replacing 'packagename' with your package name)::
+#. **Copy the template over your package**: (replacing 'packagename' with your package name)::
 
    $ cp -r /tmp/packagename ./
```

Comment:
Wait but if you do that won't it make a folder with the name of the package inside the repo? Wouldn't it make more sense to do an rsync?

Reply:
I changed this line to work (I tested it) and not use rsync because it's possible people wouldn't have that installed.